packages/cashier/src/components/crypto-transactions-history/__tests__/crypto-transactions-cancel-modal.spec.tsx

```diff
@@ -9,7 +11,21 @@ jest.mock('Stores/connect.js', () => ({
 }));
 
 describe('<CryptoTransactionsCancelModal />', () => {
-    let modal_root_el;
+    let modal_root_el, mockRootStore;
```

Comment:
```suggestion
    let modal_root_el, mockRootStore: DeepPartial<TRootStore>;
```
@vinu-deriv Same in other test files.